specification/indices/exists_index_template/IndicesExistsIndexTemplateResponse.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Response codes:
+ * 200: Indicates all specified index templates exist.
+ * 404: Indicates one or more specified index templates do not exist.
+ */
+export class Response {
+  body: {}
```

Comment:
This is an `HEAD` API, and there is no body, you should use [`Void`](https://github.com/elastic/elastic-client-generator/blob/master/docs/modeling-guide.md#void-value) :)

Reply:
See [here](https://github.com/elastic/elastic-client-generator/blob/master/specification/_global/exists/DocumentExistsResponse.ts).

Reply:
replaced all occurrences of `body: {}` with https://github.com/elastic/elastic-client-generator/pull/408/commits/ce6a15b9528c9521700ba04ce07add2f67248c10